src/components/common/AddEditOrgPerson.vue

```diff
@@ -115,13 +115,10 @@
 
             <!-- Roles -->
             <article class="mt-2">
-              <div class="font-weight-bold">{{ getPeopleAndRolesResource.rolesTitle || 'Roles' }}</div>
-              <div v-if="getPeopleAndRolesResource.rolesSubtitle">
```

Comment:
Don't need to handle this property in this "IA" org-person component.

This is moved to the "Registration" org-person component instead.